jwst/tests_nightly/general/miri/test_miri_ifu_wcs.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pytest
+
+import numpy as  np
+from numpy.testing import utils
```

Comment:
This won't work with Numpy 1.15+. Replace all `from numpy.testing import utils` with simply `from numpy.testing import your_assert_function`. Same deal with `from numpy.testing import decorators`, if you use it.